plugins/services/src/js/components/DeclinedOffersTable.js

```diff
@@ -292,11 +293,13 @@ class RejectedOffersTable extends React.Component {
     const {offers} = this.props;
 
     return (
-      <Table className="table table-simple table-header-nowrap table-break-word flush-bottom"
-        colGroup={this.getColGroup()}
-        columns={this.getColumns()}
-        data={offers}
-        sortBy={{prop: 'timestamp', order: 'desc'}} />
+      <MountService.Mount type="DeclinedOffersTable">
```

Comment:
Wonder if it makes sense to choose a less specific *types* (think CSS class), as the intention seems to mount the same type of component here as well as in the service page in case of errors. It might also worth thinking about ways to add a general page level error mount.